.travis.yml

```diff
@@ -56,8 +61,15 @@ before_script:
   - chmod +x ./cc-test-reporter
   - ./cc-test-reporter before-build
 
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH=$HOME/.yarn/bin:$PATH
+
+install:
+  - yarn install --no-lockfile --non-interactive
+
 script:
-  - COVERAGE=true node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
+  - COVERAGE=true node_modules/.bin/ember try:one $EMBER_TRY_SCENARIO
```

Comment:
What was this removed? Is it not needed?
From what I understand --skip-cleanup also makes CI faster no?

Reply:
If remember correctly then it was no longer in the default travis.yml blueprint. So I made it match the diff from “ember init”.